addon/components/frost-gauge.js

```diff
@@ -0,0 +1,169 @@
+import Ember from 'ember'
+const {isPresent} = Ember
+import computed, {readOnly} from 'ember-computed-decorators'
+import {Component} from 'ember-frost-core'
+import {PropTypes} from 'ember-prop-types'
+import layout from '../templates/components/frost-gauge'
+
+const directions = {
+  CLOCKWISE: 1,
+  COUNTERCLOCKWISE: -1
+}
+
+export default Component.extend({
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
+
+  attributeBindings: ['style'],
+  layout,
+
+  // == PropTypes =============================================================
+
+  propTypes: {
+    // required
+    data: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
As discussed I think that the only required property is data. We can compute max and have defaults for min and size